RocketChat/Rocket.Chat.ReactNative
# Chore: Evaluate ThreadMessagesView - TypeScript

## Proposed changes
Evaluate ThreadMessagesView

## Issue(s)	


## How to test or reproduce
Everything related to Threads.

E.G: 
* Create new threads; 
* send messages on thread;
* Check old messages on existed threads;
* etc;

## Screenshots

## Types of changes




## Checklist



## Further comments
